Shared/Models/Migrations/DataMigrationManager.swift

```diff
@@ -10,41 +10,43 @@ import CoreData
 import Foundation
 
 class DataMigrationManager {
```

Comment:
```suggestion
final class DataMigrationManager {
```
To avoid dynamic dispatch